_layouts/conference.html

```diff
@@ -0,0 +1,67 @@
+---
+layout: base
+---
+
+<div class="about-page grid-wrapper conference">
+  <div class="grid__item width-12-12">
+    <h1>Wild<strong>Fly</strong> Mini Conference</h1>
+    <h2 class="page-subtitle">A dedicated conference for the community with topics around Wild<strong>Fly</strong>.</h2>
+  </div>
+
+  <div class="grid__item width-12-12 sec-title-red">
+    <h2>What you need to kow</h2>
+    <dl class="c-when-and-where">
+      <dt>Date</dt>
+      <dd>Wednesday, March 6, 2024</dd>
+      <dt>Time</dt>
+      <dd>14:00 - 18:00 UTC</dd>
+      <dt>Location</dt>
+      <dd>
+        Virtually on YouTube Live: <a href="{{site.baseurl}}/conference" target="_blank">https://link-to-you-tube</a>
+      </dd>
+    </dl>
+    <p style="margin-bottom: 0">Registration is not required, but you can let us know that you <a href="{{site.baseurl}}/conference" target="_blank">plan to attend</a>.</p>
+  </div>
+
+  <div class="grid__item width-12-12 sec-title-yellow">
+    <h2>What you can expect</h2>
+    <p>The conference includes four sessions with topics for both beginners and advanced WildFly users. There will be an opening and closing ceremony and a 15-minute break between the second and third session. We want all sessions to be interactive, and you can ask questions at any time. Remember this is a conference for you - the WildFly community!</p>
+    <p>All times are in UTC.</p>
```

Comment:
This is just more of a thought. Should we make this bold? Another option would be to add `(UTC)` after each time.

Reply:
+1 for adding UTC after each time as well.

Reply:
This is just a random idea, but what about a popover that shows the date in the users current time? A quick example would look something like this https://github.com/hpehl/wildfly.org/compare/conference...jamezp:wildfly.org:date-conversion. Note the position of the popover is kind of odd so we'd want to sort that out, but just an idea :)

Reply:
Adding "UTC" to every time breaks the layout - especially on mobile devices. Instead I use @jamezp suggestion and show the local time when clicking on the time.

Reply:
I've updated the preview at https://hpehl.github.io/wildfly.org/
WDYT?

Reply:
That's pretty cool! For some reason, when I click on it though, I'm still seeing UTC time, e.g., for the Welcome session, I'm seeing 2:00-2:15pm.

Reply:
There was an issue in the JS code. Should be fixed now. @fjuma can you verify?

Reply:
Perfect, that works for me now. Thanks @hpehl!